### DISCUSSION TITLE
Renaming Rikaichamp

### DISCUSSION THREAD

**Discussion by birtles** [birchill/10ten-ja-reader#648] [General]
Hi all,

I wanted to let you know about a change I'd like to make. I've been working on a few side projects that are somewhat related to Rikaichamp for several years now. They're still several months (years?) away from being released but it's probably going to make more sense to keep them under one brand/naming scheme.

Rather than rename everything later, I thought I'd get started on renaming Rikaichamp now and then add the other pieces later.

So, I hope you don't mind if I rename Rikaichamp to "10ten Japanese reader".

Along with renaming I'd like to update the colors a little including adding a new default theme that is (boring) gray but which respects the user's dark-theme preference.

I personally use the blue/purple theme but I actually think it's kind of hideous. Obviously that option will still remain although I might tweak the colors a little.

I'm also still in two minds about whether to keep this repo under my personal account "birtles" or whether to move it to "birchill". Birchill is the company I set up to work on these things so it makes sense to move it there but I don't want this project to be seen as a corporate project. A lot of people have contributed to this project and I want it to continue to be a community project.

Let me know if you have any concerns or suggestions!

**Comment by L3m0Nate** [comment]
I appreciate a lot the page you made explaining the changes (just had it pop up), it is clear and concise. Also, I won't need measurements afaik but it's a pretty clever feature that I'm sure will benefit a lot of people! As for the theme, I still have a preference for the blue theme myself, thanks for making it prettier btw! ✨ 

As for the move to birchill (or not), I can see it's already displayed in your repositories over there, so that seems optional? I personally like the idea of keeping it on birtles for the same reason you mentioned, but it's really only a preference and I'm not a contributor myself.

Thank you and everyone else for your work.

**Reply by birtles** [reply]
> I appreciate a lot the page you made explaining the changes (just had it pop up), it is clear and concise. Also, I won't need measurements afaik but it's a pretty clever feature that I'm sure will benefit a lot of people! As for the theme, I still have a preference for the blue theme myself, thanks for making it prettier btw! ✨

Thank you so much! I'm particularly glad you like the improvements to the blue theme (I was afraid some people might be attached to the purple version).

> As for the move to birchill (or not), I can see it's already displayed in your repositories over there, so that seems optional? I personally like the idea of keeping it on birtles for the same reason you mentioned, but it's really only a preference and I'm not a contributor myself.

Right, it's not strictly necessary but it makes it a little easier for me. I've moved it to birchill so I guess it can stay there for now.

> Thank you and everyone else for your work.

Thank you for your very encouraging feedback!
